assets/MatDefs/LightingSprite.j3md

```diff
@@ -133,11 +138,11 @@ MaterialDef Phong Lighting {
         Float ExpSqFog
     }
 
- Technique {
+    Technique {
         LightMode SinglePass
 
-        VertexShader GLSL310 GLSL300 GLSL100 GLSL150:   Shaders/SPLightingSprite.vert
-        FragmentShader GLSL310 GLSL300 GLSL100 GLSL150: Shaders/SPLightingSprite.frag
+        VertexShader   GLSL310 GLSL300 GLSL150: Shaders/LightingSprite.vert

```

Comment:
This change will disable single pass light mode for good?

Reply:
It consolidates it into 1 shader. 
There's no lighting for those sprites anyway.